packages/config/src/layer2s/optimism.ts

```diff
@@ -349,5 +349,11 @@ export const optimism: Layer2 = {
       date: '2022-05-31T00:00:00Z',
       description: 'The first round of OP token airdrop.',
     },
+    {
+      name: 'EVM Equivalence is live',
+      link: 'https://twitter.com/optimismfnd/status/1458953238867165192?s=21&t=cQ0NPREYt-u1rP7OiPFKUg',
+      date: '2021-11-12T00:00:00Z',
+      description: 'Network upgrade completed.',
```

Comment:
I think we should clearly state here that fraud proofs were removed then. @luke7211 do you remember if this is how it really happened?

Reply:
Milestones for Optimism:
- Jan 2021 - Mainnet Soft launch - only selected contract like Synthetix and Uniswap are available  (https://medium.com/ethereum-optimism/mainnet-soft-launch-7cacc0143cd5) 
- Aug 2021 - Community launch - any smart contract, after being scanned by Optimism team, will be allowed (https://medium.com/ethereum-optimism/community-launch-7c9a2a9d3e84) 
- Nov 2021 - Network upgrade to OVM 2.0 and removal of fraud-proof system
- Dec 2021: Removal of a whitelist for developers - anyone can deploy anything: https://medium.com/ethereum-optimism/all-gas-no-brakes-8b0f32afd466
- Jan 2020: Introduction of call data compression (https://twitter.com/optimismFND/status/1479840872954875905?s=20&t=ftzAp4UlWICnUsfDI9YTIw)